File: 22494data/FBA.jl
using JSON, SparseArrays
open("Recon3D.json") do f
    dict = JSON.parse(f);
    data = dict["reactions"];
    temp_name = [(data[i])["name"] for i in 1:length(data)];
    b = findall(temp_name .== "Generic Human Biomass Reaction")[1];
    order = [1:(b-1); (b+1):(length(temp_name)); b];
    global name = [(data[i])["name"] for i in order];
    global lower_bound = [(data[i])["lower_bound"] for i in order];
    global upper_bound = [(data[i])["upper_bound"] for i in order];
    global subsystem = [(data[i])["subsystem"] for i in order];
    metabolites = [(data[i])["metabolites"] for i in order];
    id = [((dict["metabolites"])[i])["id"] for i in 1:length(dict["metabolites"])];
    S = Matrix{Float64}(undef, length(id), length(metabolites));
    for i in 1:length(metabolites)
        for j in 1:length(id)
            S[j, i] = get(metabolites[i], id[j], 0)
        end
    end
    global S = sparse(S); 
end

File: 22494data/FBA.py
import json
import numpy as np
with open("Recon3D.json", "r") as file_handle:
    dictionary = json.load(file_handle);
    data = dictionary["reactions"];
    temp_name = [(data[i])["name"] for i in range(len(data))];
    b = [i for i in range(len(data)) if temp_name[i] == "Generic Human Biomass Reaction"];
    b = b[0];
    order = [i for j in (range(b), range(b+1, len(data)), range(b,b+1)) for i in j];
    name = [(data[i])["name"] for i in order];
    lower_bound = [(data[i])["lower_bound"] for i in order];
    upper_bound = [(data[i])["upper_bound"] for i in order];
    subsystem = [(data[i])["subsystem"] for i in order];
    metabolites = [(data[i])["metabolites"] for i in order];
    id = [((dictionary["metabolites"])[i])["id"] for i in range(len(dictionary["metabolites"]))];
    S = np.zeros((len(id), len(metabolites)));
    for i in range(len(metabolites)):
        for j in range(len(id)):
            if id[j] in metabolites[i].keys():
                S[j, i] = metabolites[i][id[j]];

